tools/istio-iptables/pkg/log/nflog.go

```diff
@@ -0,0 +1,117 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"context"
+	golog "log"
+	"net"
+	"os"
+	"time"
+
+	"github.com/florianl/go-nflog/v2"
```

Comment:
New depenency and logger ? Or is this something specific to nflog ?

Reply:
its an nflog specific thing. Basically nflog is serving some logs over a uds socket so the library is just a bit of code to read from this. We use the standard Istio logger to write out the data read by the library